packages/terra-worklist-data-grid/tests/jest/Cell.test.jsx

```diff
@@ -0,0 +1,195 @@
+    const wrapper = shallowWithIntl(
+      <Cell
+        rowId="RowID"
+        columnId="ColumnId"
+        rowIndex={1}
+        columnIndex={2}
+        key="key"
+        isSelectable
+        onCellSelect={jest.fn}
+      >
+        Data in cell
+      </Cell>,
+    ).dive();
+
+    const selectableCell = wrapper.find('td.selectable');
+    expect(selectableCell).toHaveLength(1); // Cell should have been styled selectable.
+
+    const cellContent = selectableCell.find('div');
+    expect(cellContent).toHaveLength(1);
+    expect(cellContent.text()).toEqual('Data in cell');
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('verifies that the cell rendered is marked blank when the cell content is empty', () => {
+    const wrapper = shallowWithIntl(
+      <Cell
+        rowId="RowID"
+        columnId="ColumnId"
+        rowIndex={1}
+        columnIndex={2}
+        key="key"
+        onCellSelect={jest.fn}
+      />,
+    ).dive();
+
+    const blankCell = wrapper.find('td.blank');
+    expect(blankCell).toHaveLength(1);
+
+    const cellContent = blankCell.find('div');
+    expect(cellContent).toHaveLength(0);
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('verifies that a selected cell has the correct styles when isSelected prop is true', () => {
+    const wrapper = shallowWithIntl(
+      <Cell
+        rowId="RowID"
+        columnId="ColumnId"
+        ariaLabel="Some Label Here"
+        rowIndex={1}
+        columnIndex={2}
+        key="key"
+        isSelected
+        onCellSelect={jest.fn}
+      >
+        Data in cell
+      </Cell>,
+    ).dive();
+
+    const selectedCell = wrapper.find('td.selected');
+    expect(selectedCell).toHaveLength(1); // Cell should have been styled selected.
+
+    expect(wrapper.find('td').prop('aria-selected')).toBe(true);
+    expect(wrapper.find('td').prop('aria-label')).toBe('Some Label Here');
+
+    const cellContent = selectedCell.find('div');
+    expect(cellContent).toHaveLength(1);
+    expect(cellContent.text()).toEqual('Data in cell');
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('verifies that a cell has the correct styles and no content when isMasked prop is true', () => {
+    const wrapper = shallowWithIntl(
+      <Cell
+        rowId="RowID"
+        columnId="ColumnId"
+        ariaLabel="Some Label Here"
+        rowIndex={1}
+        columnIndex={2}
+        key="key"
+        isMasked
+        onCellSelect={jest.fn}
+      >
+        Data in cell
+      </Cell>,
+    ).dive();
+
+    const maskedCell = wrapper.find('td.masked');
+    expect(maskedCell).toHaveLength(1);
+
+    expect(wrapper.find('td').prop('aria-label')).toBe('Terra.worklistDataGrid.maskedCell');
+
+    const cellContent = maskedCell.find('div');
+    expect(cellContent).toHaveLength(0); // Masked cell should have no content.
+
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('renders cell as mask only even if cell is selectable and selected', () => {
```

Comment:
is this a typo?
```suggestion
  it('renders cell as mask even if cell is selectable and selected', () => {
```

Reply:
No but I changed the text to make it read more clearly.